tests/blockchain/test_blockchain.py

```diff
@@ -3131,18 +3131,20 @@ async def test_basic_reorg(self, empty_blockchain, bt):
         assert b.get_peak().height == 16
 
     @pytest.mark.asyncio
-    async def test_long_reorg(self, empty_blockchain, default_1500_blocks, test_long_reorg_blocks, bt):
+    async def test_long_reorg(self, empty_blockchain, default_10000_blocks, test_long_reorg_blocks, bt):
         # Reorg longer than a difficulty adjustment
         # Also tests higher weight chain but lower height
         b = empty_blockchain
-        num_blocks_chain_1 = 3 * bt.constants.EPOCH_BLOCKS + bt.constants.MAX_SUB_SLOT_BLOCKS + 10
-        num_blocks_chain_2_start = bt.constants.EPOCH_BLOCKS - 20
+        num_blocks_chain_1 = 2000
```

Comment:
looks like this could change the test since the relation to the constants is not enforced anymore

Reply:
My understanding is that the test wants to make sure there are at least 3 epochs before the fork. I don't think the exact block height matters so much other than that. Do you think there's something subtle being tested by this exact fork height?